src/GraphQL/FieldHelper/AssetFieldHelper.php

```diff
@@ -23,6 +23,47 @@
 
 class AssetFieldHelper extends AbstractFieldHelper
 {
+    public function getVideoThumbnail(Asset\Video $asset, string | Video\Thumbnail\Config $thumbNailConfig, string $thumbNailFormat = null): mixed
+    {
+        if (isset($thumbNailFormat) && $thumbNailFormat !== 'image') {
+            $value = $asset->getThumbnail($thumbNailConfig);
+            if ($value) {
+                $formats = $value['formats'] ?? [];
+                $format = $formats[$thumbNailFormat] ?? null;
+                if ($format) {
+                    return $format;
+                }
+            }
+        } else {
+            return $asset->getImageThumbnail($thumbNailConfig);
+        }
+
+        return null;
+    }
+
+    public function getImageDocumentThumbnail(Asset $asset, string | Image\Thumbnail\Config $thumbNailConfig, string $thumbNailFormat = null): mixed
+    {
+        if ($asset instanceof Asset\Document || $asset instanceof Asset\Video) {
+            $thumb = $asset->getImageThumbnail($thumbNailConfig);
+        } else {
```

Comment:
This should fix the new PhpStan error, right?
```suggestion
        } elseif ($asset instanceof Asset\Image) {
```

Reply:
Yes, this sould fix the error, but then $thumb would be undefined. I´ll add a check with the next commit, thanks!